tests/models/rnafusion/test_rnafusion_sample.py

```diff
@@ -1,97 +1,120 @@
-from typing import List
+from pathlib import Path
 
 import pytest
 from pydantic.v1 import ValidationError
-from pydantic.v1 import ValidationError as PydanticValidationError
 
 from cg.exc import SampleSheetError
 from cg.models.rnafusion.rnafusion import RnafusionSample
 
 
 def test_instantiate_rnafusion_sample(
-    rnafusion_sample: str,
-    rnafusion_fastq_forward: List[str],
-    rnafusion_fastq_reverse_same_length: List[str],
-    rnafusion_strandedness_acceptable: str,
+    sample_name: str,
+    fastq_forward_read_path: Path,
+    fastq_reverse_read_path: Path,
+    strandedness: str,
 ):
     """
-    Tests rnafusion sample.
+    Tests Rnafusion sample with correct values.
     """
     # GIVEN a sample with fastq files and strandedness
 
     # WHEN instantiating a rnafusion sample object
     rnafusion_sample_object = RnafusionSample(
-        sample=rnafusion_sample,
-        fastq_forward=rnafusion_fastq_forward,
-        fastq_reverse=rnafusion_fastq_reverse_same_length,
-        strandedness=rnafusion_strandedness_acceptable,
+        name=sample_name,
+        fastq_forward_read_paths=[fastq_forward_read_path, fastq_forward_read_path],
+        fastq_reverse_read_paths=[fastq_reverse_read_path, fastq_reverse_read_path],
+        strandedness=strandedness,
     )
 
     # THEN assert that it was successfully created
     assert isinstance(rnafusion_sample_object, RnafusionSample)
 
 
 def test_fastq_forward_reverse_different_length(
```

Comment:
```suggestion
def test_instantiate_rnafusion_sample_fastq_forward_reverse_different_length(
```

Reply:
I'm not a big fan of such a long name. Renaming to `test_incomplete_fastq_file_pairs`